deployment/ansible/roles/model-my-watershed.spark-jobserver/templates/spark-jobserver.conf.j2

```diff
@@ -38,4 +38,9 @@ spark {
 # spray #
 #########
 
-spray.can.server.parsing.max-content-length = 250m
+spray.can.server {
+        parsing.max-content-length = 250m
+        # Match the backend's timeout set here src/mmw/apps/modeling/tasks.py
+        idle-timeout = 42s
```

Comment:
Took a bit of spelunking to figure out why SJS wouldn't start. Finally discovered that `idle-timeout` must always be greater than the value of `request-timeout`:

```
[JobServer-akka.actor.default-dispatcher-4] ERROR akka.actor.OneForOneStrategy - requirement failed: idle-timeout must be > request-timeout (if the latter is not 'infinite')
akka.actor.ActorInitializationException: exception during creation
	at akka.actor.ActorInitializationException$.apply(Actor.scala:166)
	at akka.actor.ActorCell.create(ActorCell.scala:596)
	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:456)
	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:478)
	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:263)
	at akka.dispatch.Mailbox.run(Mailbox.scala:219)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:397)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.lang.IllegalArgumentException: requirement failed: idle-timeout must be > request-timeout (if the latter is not 'infinite')
	at scala.Predef$.require(Predef.scala:233)
	at spray.can.server.ServerSettings$Timeouts.<init>(ServerSettings.scala:75)
	at spray.can.server.ServerSettings$.fromSubConfig(ServerSettings.scala:86)
	at spray.can.server.ServerSettings$.fromSubConfig(ServerSettings.scala:60)
	at spray.util.SettingsCompanion.apply(SettingsCompanion.scala:50)
	at spray.util.SettingsCompanion$$anonfun$apply$1.apply(SettingsCompanion.scala:36)
	at scala.collection.MapLike$class.getOrElse(MapLike.scala:128)
	at scala.collection.AbstractMap.getOrElse(Map.scala:58)
	at spray.util.SettingsCompanion.apply(SettingsCompanion.scala:35)
	at spray.can.server.HttpListener$$anonfun$1.apply(HttpListener.scala:34)
	at spray.can.server.HttpListener$$anonfun$1.apply(HttpListener.scala:34)
	at scala.Option.getOrElse(Option.scala:120)
	at spray.can.server.HttpListener.<init>(HttpListener.scala:34)
	at spray.can.HttpManager.newHttpListener(HttpManager.scala:84)
	at spray.can.HttpManager$$anonfun$receive$1$$anonfun$applyOrElse$1.apply(HttpManager.scala:76)
	at spray.can.HttpManager$$anonfun$receive$1$$anonfun$applyOrElse$1.apply(HttpManager.scala:76)
	at akka.actor.TypedCreatorFunctionConsumer.produce(Props.scala:343)
	at akka.actor.Props.newActor(Props.scala:252)
	at akka.actor.ActorCell.newActor(ActorCell.scala:552)
	at akka.actor.ActorCell.create(ActorCell.scala:578)
	... 9 more
```

Updating it to `45s` did the trick.

Also consider using 2 spaces for indentation, as the rest of the file does. Optionally right align the values as shown in the documentation http://spray.io/documentation/1.2.4/spray-can/configuration/

```yaml
spray.can.server {
  parsing.max-content-length = 250m

  # Match the backend's timeout set here src/mmw/apps/modeling/tasks.py
  # idle-timeout _must be_ greater than request-timeout
  idle-timeout    = 45s
  request-timeout = 42s
}
```